internal/jujuapi/applicationoffers.go

```diff
@@ -178,11 +178,17 @@ func (r *controllerRoot) modifyOfferAccess(ctx context.Context, change jujuparam
 		if err := r.jimm.PermissionManager().GrantOfferAccess(ctx, r.user, change.OfferURL, ut, change.Access); err != nil {
 			return errors.E(op, err)
 		}
+		if err := r.jimm.JujuManager().GrantOfferAccessOnController(ctx, r.user, ut, change.OfferURL, change.Access); err != nil {
```

Comment:
Can you put a comment explaining that this is for pre Juju 3.6.6 controllers and why it works.

Reply:
done